lib/plotutil.py

```diff
@@ -114,13 +117,15 @@ def plot_line(arrays, color_name, xlabel, ylabel, theme, gui, line_style, file_p
 
     plt.grid(True)
 
+    if file_path != "":
+    	plt.savefig(file_path)
+        
```

Comment:
@djmgit I have made necessary changes please review PR. I have also tested it I think it works fine. :)

![screenshot_1](https://user-images.githubusercontent.com/16613832/34453972-6381b97c-ed86-11e7-93cc-295d02f8e72a.png)


![screenshot_2](https://user-images.githubusercontent.com/16613832/34453973-6617ab7e-ed86-11e7-99f8-70232753e408.png)